defaults/main.yml

```diff
@@ -1,7 +1,33 @@
 ---
-# defaults file for backups-role
 backups_role_restic_version: '0.9.3'
-backups_role_path: '/opt/backups'
-backups_role_config_paths: []
+backups_role_script_path: '/opt'
+backups_role_tmp_path: '/tmp/backups'
+backups_role_config_paths:
+    - '/etc'
+    - '/root'
+    - '/usr/local/etc'
 backups_role_assets_paths: []
-backups_role_cron_job_at: '5 10 * * *'
+backups_role_cron_job_at: '45  3 * * *'
+
+backups_role_restic_repo_name: "{{ ansible_hostname }}"
+
+backups_role_user_name: 'backups'
+backups_role_user_group: 'backups'
+backups_role_sudoers_cmd_pattern: '/bin/tar -czvf *'
+
+# unused
+#backups_role_restic_user: "{{ backups_role_user_name }}"
+# NOTE. Only last command of the && chain will use this log.
+# See *fixme* below to fix it
+backups_role_cron_stdout_file: '/var/log/cron.d/restic-stdout.log'
+backups_role_cron_stderr_file: '/var/log/cron.d/restic-stderr.log'
+
+# read only user
+backups_role_postgresql_user_name: "{{ backups_role_user_name }}"
+# manager user
+postgresql_user: "postgres"
+# NOTE Please set this in a vault. Using a known default password or void one is unsafe.
```

Comment:
Maybe we can have an unencrypted `secrets.yml.example` file with all the variables that needs to be encrypted.

Like https://github.com/coopdevs/timeoverflow-provisioning/blob/master/inventory/host_vars/local.timeoverflow.org/secrets.yml but with `.example` as extension to avoid to get it used by Ansible.

Reply:
Oh, good idea! Where could we fit the `secrets.yml.example` in? Also in defaults?

Reply:
I have placed it in defaults, feel free to move it or propose other locations d2057b3e1791d66dba8ef183f55954df01c4c7e7